packages/firestore/test/unit/local/index_free_query_engine.test.ts

```diff
@@ -264,7 +264,28 @@ describe('IndexFreeQueryEngine', () => {
   it('does not use initial results for limit query with document removal', async () => {
     const query = Query.atPath(path('coll'))
       .addFilter(filter('matches', '==', true))
-      .withLimit(1);
+      .withLimitToFirst(1);
```

Comment:
Instead of duplicating the limit() tests, what do you think about making one of them use `limitToLast()` and keeping just `limitToFirst()` for the rest of them?

Reply:
Done.